_posts/en/newsletters/2023-10-11-newsletter.md

```diff
@@ -0,0 +1,189 @@
+---
+title: 'Bitcoin Optech Newsletter #272'
+permalink: /en/newsletters/2023/10/11/
+name: 2023-10-11-newsletter
+slug: 2023-10-11-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME:harding
+
+## News
+
+- **Specification for `OP_TXHASH` proposed:** Steven Roose [posted][roose
+  txhash] to the Bitcoin-Dev mailing list a [draft BIP][bips #1500] for
+  a new `OP_TXHASH` opcode.  The idea of behind this opcode has been
+  discussed before (see [Newsletter #185][news185 txhash]) but this is
+  the first specification of the idea.  In addition to describing
+  exactly how the opcode would work, it also looks at mitigating some
+  potential downsides, such as full nodes potentially needing to hash up
+  to several megabytes of data every time the opcode is invoked.
+  Roose's draft includes a sample implementation of the opcode.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review
+Club][] meeting, highlighting some of the important questions and
+answers.  Click on a question below to see a summary of the answer from
+the meeting.*
+
+[util: Type-safe transaction identifiers][review club 28107] is a PR
+by Niklas Gögge (dergoegge) that improves type safety by introducing
+separate types for `txid` (the transaction identifier or hash that
+doesn't include the segwit witness data) and `wtxid` (same but includes
+the witness data), rather than both being represented by a `uint256`
+(a 256-bit integer, which can contain a SHA256 hash). This PR
+should have no operational effect; it aims to prevent future
+programming errors in which one kind of transaction ID is used where
+the other was intended. Such errors will be detected at compile time.
+
+To minimize disruption and ease review, these new types will initially
+be used in only one area of the code (the transaction "orphanage");
+future PRs will deploy the new types to other areas of the codebase.
+
+{% include functions/details-list.md
+  q0="What does it mean for a transaction identifier to be type-safe?
+      Why is that important or helpful? Are there any downsides?"
+  a0="A transaction identifier is considered type-safe when it undergoes
+      rigorous type-checking during compile-time, guaranteeing that
```

Comment:
Seems to be a copy-paste of one of the answers?

Reply:
It is... but it seemed really good. It does seem a little, what's the word, formal or formulaic, so I'll work on it, thanks.